### PR TITLE
Serve content from Django as fallback

### PR DESCRIPTION
**Pull request by kachkaev**
Relevant Next.js docs:
https://nextjs.org/docs/api-reference/next.config.js/rewrites#incremental-adoption-of-nextjs

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Place the below variables in .env.local
+# DATABASE_URL="postgres://user:password@host:5432/db"
+# AUTH0_SECRET='replace-with-your-own-secret-generated-with-openssl'
+# AUTH0_BASE_URL='http://localhost:3000'
+# AUTH0_ISSUER_BASE_URL='https://{DOMAIN}'
+# AUTH0_CLIENT_ID='{CLIENT_ID}'
+# AUTH0_CLIENT_SECRET='{CLIENT_SECRET}'
+# AUTH0_AUDIENCE=
+# AUTH0_SCOPE='openid profile'
```
Comment (kachkaev) [inline .env:11]: Extracted into https://github.com/dtpstat/website/pull/29

Comment (kachkaev) [reply .env]: ✅

File: .env [hunk 2]
```diff
@@ -1,3 +1,6 @@
+DJANGO_BASE_URL=https://dtp-stat.ru
+DJANGO_CONTENT_FALLBACK=false
```
Comment (kachkaev) [inline .env:2]: It might be best to keep this feature off during local dev by default. It helps detect what assets are missing.

File: .env [hunk 3]
```diff
@@ -1,3 +1,4 @@
 AUTH0_SCOPE=openid profile
 
 DJANGO_BASE_URL=https://dtp-stat.ru
+DJANGO_CONTENT_FALLBACK=false
```
Comment (kachkaev) [inline .env:4]: I suggest us to keep this feature off by default. It helps detect what assets are missing during local dev. We also increase each other’s awareness of the feature by having to enable it by hand.

Comment (kachkaev) [reply .env:4]: I’ve set `DJANGO_CONTENT_FALLBACK` to `true` on Netlify.

File: README.md [hunk 4]
```diff
@@ -30,11 +30,37 @@ Future frontend of https://dtp-stat.ru written in Next.js.
...
+
 ### Content management
 
 ```ini
-## (optional) Pausing comments
-COMMENTS_ARE_PAUSED=true
+COMMENTS_ARE_PAUSED={true | false}
+```
+
+### Error tracking
```
Comment (kachkaev) [inline README.md:54]: This section was outside _Environment variables_, I moved it in.

**Review by sergous — APPROVED** [review]

**Review by sergous — APPROVED** [review]

**Pull request merged**